ts/parts-more/Polar.ts

```diff
@@ -260,14 +261,14 @@ seriesProto.toXY = function (
         xAxis = this.xAxis,
         yAxis = this.yAxis,
         plotX = point.plotX,
-        plotY = point.plotY as any,
+        plotY = point.plotY,
         series = point.series,
         inverted = chart.inverted,
         clientX;
 
     // Corrected y position of inverted series other than column
     if (inverted && series && !series.isRadialBar) {
-        plotY = (point.plotY as any) = yAxis.translate(point.y as any);
+        plotY = point.plotY = yAxis.translate(point.y as any) || 0;
```

Comment:
what is the reason of using `as any` ?

Reply:
I have added an additional check to see if the point.y is a number.